masterlist.yaml

```
@@ -8820,6 +8820,19 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507'
```

Links should have trailing slashes for consistency & convenience:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507/'
```

---

😅